app/controllers/wizard/steps/country_of_origin_controller.rb

```diff
@@ -18,6 +18,23 @@ def permitted_params
           :country_of_origin,
         )
       end
+
+      def opts
+        return opts_for_ni_to_gb_route if answer == 'GB' && user_session.import_destination == 'XI'
```

Comment:
Makes sense to me. I'm not a fan of the step having a different constructor from the base step but I guess we have to pick a poison somewhere.

Reply:
Yeah I think it's ok.